src/toxcore/tcp/server/server.rs

```diff
@@ -365,6 +368,25 @@ impl Server {
             Box::new( future::ok(()) )
         }
     }
+    /** Send pings to all connected clients
```

Comment:
Add note that this method removes timed out clients, something like this:
`Send pings to all connected clients and terminate all timed out clients.`